lib/matplotlib/font_manager.py

```diff
@@ -1177,25 +1177,39 @@ def score_size(self, size1, size2):
     def findfont(self, prop, fontext='ttf', directory=None,
                  fallback_to_default=True, rebuild_if_missing=True):
         """
-        Search the font list for the font that most closely matches
-        the :class:`FontProperties` *prop*.
-
-        :meth:`findfont` performs a nearest neighbor search.  Each
-        font is given a similarity score to the target font
-        properties.  The first font with the highest score is
-        returned.  If no matches below a certain threshold are found,
-        the default font (usually DejaVu Sans) is returned.
-
-        `directory`, is specified, will only return fonts from the
-        given directory (or subdirectory of that directory).
+        Find a font that most closely matches the given font properties.
+
+        Parameters
+        ----------
+        prop : `~matplotlib.fontmanager.FontProperties`
```

Comment:
Apparently, this supports strings as well. I can do `findfont('Calibri')` or `findfont('Calibri:bold:')` what is the correct terminology for the string representation?

Reply:
it's a "fontconfig pattern" (mentioned elsewhere in the module).